### PR TITLE
[simple_planning_simulator] add readme

### PR DESCRIPTION
**Pull request by TakaHoribe**
Add readme for `simple_planning_simulator` node.

### CODE REVIEW

File: simulator/simple_planning_simulator/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+This node simulates the vehicle motion for a vehicle command in 2D using a simple vehicle model.
+
+## Interface
+
+**input**
+ - /control/vehicle_cmd [`autoware_vehicle_msgs/VehicleCommand`] : target command to drive a vehicle.
+ - /initialpose [`geometry_msgs/PoseWithCovarianceStamped`] : for initial pose
+ - /initialtwist [`geometry_msgs/TwistStamped`] : for initial velocity
+ - /planning/scenario_planning/trajectory [`autoware_planning_msgs/Trajectory`]: for z position
+ - /vehicle/engage
```
Comment (tkimura4) [inline simulator/simple_planning_simulator/README.md:11]: /vehicle/engage
send true: the simulator (re)starts.
send false: the simulator pauses.

**Review by UMiho — DISMISSED** [review]
LGTM

File: simulator/simple_planning_simulator/README.md (hunk 1 above)
Comment (TakaHoribe) [inline simulator/simple_planning_simulator/README.md:11]:
```suggestion
 - /vehicle/engage : if true, the vehicle starts to move. if false, stops.
```

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**